Wrong resolution to Vue 2 when in Vue 3 project

I'm using `fluent-vue` v3.0.0 in a Vue 3 project. When building, I got this error:

```
❯ yarn build                                        
yarn run v1.22.17
$ vue-tsc --noEmit && vite build
node_modules/fluent-vue/index.d.ts:19:40 - error TS2709: Cannot use namespace 'Vue' as a type.

19   interface ComponentOptions<V extends Vue> {
                                          ~~~


Found 1 error.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/index.d.ts
+++ b/index.d.ts
@@ -1,26 +1,8 @@
-/* eslint-disable @typescript-eslint/prefer-ts-expect-error */
-/* eslint-disable @typescript-eslint/ban-ts-comment */
-// @ts-ignore
-import Vue from 'vue'
-// @ts-ignore
-import '@vue/runtime-core'
-import { FluentResource } from '@fluent/bundle'
-
 // #region Vue 2
 declare module 'vue/types/vue' {
   interface Vue {
     $t: (key: string, values?: Record<string, unknown>) => string
     $ta: (key: string, values?: Record<string, unknown>) => Record<string, string>
-  }
-}
-
-declare module 'vue/types/options' {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  interface ComponentOptions<V extends Vue> {
-    /**
-     * Message override for Vue component
-     */
-    fluent?: { [locale: string]: FluentResource }
   }
 }
 // #endregion
@@ -30,12 +12,6 @@ declare module '@vue/runtime-core' {
   interface ComponentCustomProperties {
     $t: (key: string, values?: Record<string, unknown>) => string
     $ta: (key: string, values?: Record<string, unknown>) => Record<string, string>
-  }
-}
-
-declare module '@vue/runtime-core' {
-  interface ComponentCustomOptions {
-    fluent?: { [locale: string]: FluentResource }
   }
 }
 // #endregion

